content/docs/get-started/azure/modify-program.md

```diff
@@ -144,7 +151,10 @@ class MyStack : Stack
             ResourceGroupName = resourceGroup.Name,
             AccountReplicationType = "LRS",
             AccountTier = "Standard",
-            EnableHttpsTrafficOnly = true
+            Tags = new Dictionary<string, string>
```

Comment:
```suggestion
            Tags =
```

No need to make a new dictionary here. Feel free to convert to one-liner if that looks better to you.